pkg/config/optimizely_config.go

```diff
@@ -0,0 +1,161 @@
+/****************************************************************************
+ * Copyright 2019, Optimizely, Inc. and contributors                        *
+ *                                                                          *
+ * Licensed under the Apache License, Version 2.0 (the "License");          *
+ * you may not use this file except in compliance with the License.         *
+ * You may obtain a copy of the License at                                  *
+ *                                                                          *
+ *    http://www.apache.org/licenses/LICENSE-2.0                            *
+ *                                                                          *
+ * Unless required by applicable law or agreed to in writing, software      *
+ * distributed under the License is distributed on an "AS IS" BASIS,        *
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. *
+ * See the License for the specific language governing permissions and      *
+ * limitations under the License.                                           *
+ ***************************************************************************/
+
+// Package entities //
+package config
+
+import "github.com/optimizely/go-sdk/pkg/entities"
+
+// OptimizelyConfig is a snapshot of the experiments and features in the project config
+type OptimizelyConfig struct {
+	Revision       string
+	ExperimentsMap map[string]OptimizelyExperiment
```

Comment:
Should these entities types be pointers? Does it matter if we return the pointer to the config?

Reply:
maps are already pointers to the underlying array already so it's ok to have it like that